pkg/server/controllers.go

```diff
@@ -693,6 +694,39 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 	})
 }
 
+func (s *Server) installAPIBinderController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	controllerName := "kcp-apibinder-controller"
+	config = rest.CopyConfig(config)
+	// Do we need a round tripper here?
+	config = rest.AddUserAgent(config, controllerName)
+
+	kcpClusterClient, err := kcpclient.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	// The CWT is coming from the YAML in Steve's PR, but what do we need out of it? Just the URL?
+	// Can we query it out of the shard?
+	//cwt, err = tenancyClient.ClusterWorkspaceTypes().Create(ctx, cwt, metav1.CreateOptions{})
```

Comment:
I'm looking at [the de-serialization](https://github.com/stevekuznetsov/kcp/blob/c9e01e95b4a97cbb4af8473844e22ebe6dedd2c9/pkg/reconciler/tenancy/initialization/apibinder/bootstrap.go#L148); where would this come from if we're not creating it from bundled YAML?

Reply:
As discussed on Slack:

- use a special reserved, hard-coded initializer name for apibindings, such as `kcp.dev/apibindings`
- modify the regex for initializer types to support either `root:*` or `kcp.dev/*`
- add the apibindings initializer in clusterworkspacetypeexists admission if the cwt spec has apibindings
- remove this initializer when the apibindings have been created